node_modules/oae-core/topnavigation/css/topnavigation.css

```diff
@@ -32,16 +32,37 @@
 
 /* Desktop resolution */
 @media (min-width : 992px) {
```

Comment:
As mentioned above, see https://github.com/oaeproject/3akai-ux/pull/3992#discussion_r36715986 for desired top nav changes